src/cpp/shared/adventure/hero.cpp

```diff
@@ -151,6 +151,14 @@ void hero::SetPrimarySkill(int skill, int amt) {
   this->primarySkills[skill] = amt;
 }
 
+void hero::ClearSS() {
+  for (int i = 0; i < 14; ++i) {
```

Comment:
Magic number

Reply:
I thought there was something off with that, but I couldn't find a matching macro for this relating to secondary skills.